plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/SwtBotAppEngineActions.java

```diff
@@ -65,9 +66,15 @@ public static IProject createNativeWebAppProject(SWTWorkbenchBot bot, String pro
       bot.textWithLabel("Java package:").setText(javaPackage);
     }
     // can take a loooong time to resolve jars (e.g. servlet-api.jar) from Maven Central
-    int libraryResolutionTimeout = 60000/* ms */;
+    int libraryResolutionTimeout = 300 * 1000/* ms */;
     SwtBotTimeoutManager.setTimeout(libraryResolutionTimeout);
-    SwtBotTestingUtilities.clickButtonAndWaitForWindowClose(bot, bot.button("Finish"));
+    try {
+      SwtBotTestingUtilities.clickButtonAndWaitForWindowClose(bot, bot.button("Finish"));
+    } catch (TimeoutException ex) {
+      System.out.println("FATAL: timed out while waiting for the wizard to close. Cannot recover. "
```

Comment:
I searched for any usage of `Logger` in tests by `$ find . -name '*Test.java' -print0 | xargs -0 grep Logger`, but there has been none. I'll skip a logger and output to stderr instead.